projects/components/src/lib/sort.directive.ts

```diff
@@ -0,0 +1,48 @@
+import { Directive, Input, OnInit } from '@angular/core';
+import { SortableDirective } from './sortable.directive';
+
+@Directive({
+    selector: '[mzSort]',
```

Comment:
Can we change this to `mzSortKey` and the other to `mzSort` please?

Reply:
done